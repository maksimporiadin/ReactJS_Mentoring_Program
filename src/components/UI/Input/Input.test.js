import 'jsdom-global/register';
import React from 'react';
import { Input } from './index';
import { shallow } from 'enzyme';

describe('Input', () => {
    let wrapper;
    const defaultProps = {
        className: 'title',
        inputConfig: {},
        onChange() {},
    };

    beforeEach(()=>{
        jest.resetAllMocks();
        wrapper = shallow(<Input {...defaultProps} />)
    });

    it('is renders', () => {
        expect(wrapper).toMatchSnapshot();
    });
});