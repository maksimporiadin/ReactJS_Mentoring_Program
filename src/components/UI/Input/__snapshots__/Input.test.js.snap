// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input is renders 1`] = `
<input
  className="title"
  onChange={[Function]}
/>
`;
